src/pages/meal-plan/meal-plan-edit-page.tsx

```
@@ -213,3 +219,222 @@ const MealPlanEditPage = () => {
+            handleModal={() => {
+              setIsModalOpen(false);
+            }}
+          />
+        )}
+
+        <button
+          className="flex items-center justify-center size-8 mb-3 cursor-pointer"
+          onClick={() => {
+            setIsOpen((prev) => !prev);
+          }}
+        >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)